src/FulfillerSelect.jsx

```diff
@@ -72,29 +72,38 @@ class FulfillerSelect extends React.Component {
             this.fetchFulfillers(this.props.accessToken);
         }
 
-        this.getRecentFulfillerIds();
+        this.getRecentFulfillerIds()
+            .then(() => {
+                if (this.state.recentFulfillerIds.length && !this.state.selectedFulfillerId) {
+                    this.setState({ selectedFulfillerId: this.state.recentFulfillerIds[0] });
+                }
+            });
     }
 
     onChange(e) {
         this.setState({
-            selectedFulfillerId: e.fulfillerId
+            selectedFulfillerId: e.fulfiller.fulfillerId
```

Comment:
Look at getOptions for an explanation of why there is `.fulfiller.` in there now.